tests/acceptance/features/apiSpaces/listSpaces.feature

```diff
@@ -526,3 +526,316 @@ Feature: List and create spaces
       | Space Admin |
       | User        |
       | User Light  |
+
+  @env-config
+  Scenario Outline: sharee lists drives after the share role Secure Viewer has been disabled (Personal Space)
+    Given user "Brian" has been created with default attributes
+    And the administrator has enabled the permissions role "Secure Viewer"
+    And user "Alice" has uploaded file with content "some content" to "textfile.txt"
+    And user "Alice" has created folder "folderToShare"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | <resource>    |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Secure Viewer |
+    And user "Brian" has a share "<resource>" synced
+    And the administrator has disabled the permissions role "Secure Viewer"
+    When user "Brian" lists all available spaces via the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON response should contain space called "Shares" owned by "Brian" and match
```

Comment:
:thinking:  why are we checking for Shares drive? it will always be listed for any user